core/arch/arm/kernel/abort.c

```diff
@@ -485,12 +485,14 @@ static bool is_vfp_fault(struct abort_info *ai __unused)
 
 static enum fault_type get_fault_type(struct abort_info *ai)
 {
-	if (is_user_exception(ai)) {
+	if (abort_is_user_exception(ai)) {
 		if (is_vfp_fault(ai))
 			return FAULT_TYPE_USER_TA_VFP;
+#ifndef CFG_WITH_PAGER
```

Comment:
if pager is enable, ALIGNMENT aborts in user TA will panic TEE core in "case CORE_MMU_FAULT_ALIGNMENT" below. whereas old code simply trapped all user TA aborts for 'TA panic' handling.